### PR TITLE
fix for #902

### PR DESCRIPTION
**Pull request by sinjan25**
fix for #902

### CODE REVIEW

File: src/mlpack/methods/policy_gradient/policy_gradient.cpp [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (zoq) [inline src/mlpack/methods/policy_gradient/policy_gradient.cpp]: Can you remove this file?

Comment (sinjan25) [inline src/mlpack/methods/policy_gradient/policy_gradient.cpp]: Okay. I was actually planning to add my implementation of policy gradient there.

Comment (sinjan25) [inline src/mlpack/methods/policy_gradient/policy_gradient.cpp]: @zoq I updated elu.hpp and removed the policy_gradient.cpp

**Comment by sam0410** [comment]
Hi @zoq !
Actually I was thinking, what is wrong in the code in my pull request? Because, it has been 3 days, u haven't reviewed it yet.

**Comment by sinjan25** [comment]
@zoq All checks have successfully passed.

**Comment by sam0410** [comment]
@sinjan25 If another person has already pulled the request which passes all the test cases, it is pointless to work on the same issue. You could take up another issue. This way more effective work can be done.

**Comment by zoq** [comment]
@sinjan25 I checked the issue and @sinjan25 was the first how said he likes to solve the issue and he was also the first who created an PR for that, so I think it's fair to merge his PR.

We are working on adding other issues to the issue tracker.

**Comment by zoq** [comment]
Issue #902 is fixed in 5215c3d, however thanks for taking a look into the issue and for the contribution.

**Pull request closed without merging**